nb-virtdata/virtdata-lang/src/main/java/io/nosqlbench/virtdata/lang/parser/VirtDataDSL.java

```diff
@@ -67,6 +62,18 @@ public static VirtDataDSL.ParseResult parse(String input) {
         }
     }
 
+    private static VirtDataParser getVirtDataParser(CodePointCharStream cstream) {
```

Comment:
are these changes required?

Reply:
Only added the error listener here to throw error when there is some syntax error otherwise if there are syntax errors there will be a line printed for it, but execution will continue.

When I added this IntelliJ was showing a yellow line saying it can be extracted into a method, so did that.
Let me know if you think we don't need this extra method, I can revert it!!!

Reply:
I think that the ".*" form of includes us not a good practice. Otherwise I think that shooky would approve.